frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -147,7 +145,7 @@ function sanitizeFilterParams(filters: Partial<FilterType>): Record<string, any>
 export const eventUsageLogic = kea<
     eventUsageLogicType<DashboardEventSource, GraphSeriesAddedSource, RecordingWatchedSource>
 >({
-    connect: [preflightLogic],
+    connect: () => [preflightLogic],
```

Comment:
Had to add this to make these files now work with webpack. This is just `undefined` when the `kea({})` line in the global scope is executed.